taxonomies/categories.txt

```diff
@@ -43772,7 +43772,7 @@ ja:クワルク
 ka:ხაჭო
 ko:크바르크
 lb:Stoffi
-lt:Varškės sūris
```

Comment:
Is "Varškės sūris" a correct name that we should add as a synonym?

Reply:
For en:Quarks,Quark it should be lt:Varškės,Varškė